image/nitf.ksy

```diff
@@ -0,0 +1,769 @@
+        -orig-id: ostaid
+        type: str
+        size: 10
+      - id: file_date_time
+        -orig-id: fdt
+        type: date_time
+      - id: file_title
+        -orig-id: ftitle
+        type: str
+        size: 80
+      - id: file_security
+        -orig-id: fclasnfo
+        type: clasnfo
+      - id: file_copy_number
+        -orig-id: fscop
+        type: str
+        size: 5
+      - id: file_num_of_copys
+        -orig-id: fscpys
+        type: str
+        size: 5
+      - id: encryption
+        -orig-id: encryp
+        type: encrypt
+      - id: file_bg_color
+        -orig-id: fbkgc
+        size: 3
+      - id: originator_name
+        -orig-id: oname
+        type: str
+        size: 24
+      - id: originator_phone
+        -orig-id: ophone
+        type: str
+        size: 18
+      - id: file_length
+        -orig-id: fl
+        type: str
+        size: 12
+      - id: file_header_length
+        -orig-id: hl
+        type: str
+        size: 6
+      - id: num_image_segments
+        -orig-id: numi
+        type: str
+        size: 3
+      - id: linfo
+        type: length_image_info
+        repeat: expr
+        repeat-expr: num_image_segments.to_i
+      - id: num_graphics_segments
+        -orig-id: nums
+        type: str
+        size: 3
+      - id: lnnfo
+        type: length_graphic_info
+        repeat: expr
+        repeat-expr: num_graphics_segments.to_i
+      - id: reserved_numx
+        -orig-id: numx
+        type: str
+        size: 3
+      - id: num_text_files
+        -orig-id: numt
+        type: str
+        size: 3
+      - id: ltnfo
+        type: length_text_info
+        repeat: expr
+        repeat-expr: num_text_files.to_i
+      - id: num_data_extension
+        -orig-id: numdes
+        type: str
+        size: 3
+      - id: ldnfo
+        type: length_data_info
+        repeat: expr
+        repeat-expr: num_data_extension.to_i
+      - id: num_reserved_extension
+        -orig-id: numres
+        type: str
+        size: 3
+      - id: lrnfo
+        type: length_reserved_info
+        repeat: expr
+        repeat-expr: num_reserved_extension.to_i
+      - id: user_defined_header
+        type: tre_header
+      - id: extended_header
+        type: tre_header
+  date_time:
+    seq:
+      - type: str
+        size: 14
+        doc: 'UTC time of image acquisition in the format CCYYMMDDhhmmss: CC century, YY last two digits of the year, MM month, DD day, hh hour, mm minute, ss second'
+  encrypt:
+    seq:
+      - type: str
+        size: 1
+  clasnfo:
+    seq:
+      - id: sclas
+        type: str
+        size: 1
+      - id: sclsy
+        type: str
+        size: 2
+      - id: scode
+        type: str
+        size: 11
+      - id: sctlh
+        type: str
+        size: 2
+      - id: srel
+        type: str
+        size: 20
+      - id: sdctp
+        type: str
+        size: 2
+      - id: sdcdt
+        type: str
+        size: 8
+      - id: sdcxm

```

Comment:
I see that you improved a lot of field namings in https://github.com/kaitai-io/kaitai_struct_formats/pull/294/commits/77c94a25c82c78c2335fb6d888102dde70a649e3, but I still feel that these (and similar ones) deserve a more meaningful `id`.

Reply:
Done

Reply:
Anything else we need to do or is this good to merge?